src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -199,12 +199,13 @@ class Filter extends Component<Props> {
     })
 
     return (
-      <React.Fragment>
-        <Sans size="2" weight="medium" color="black100" mt={0.3} mb={1}>
-          Ways to Buy
+      <Box pt={1}>
+        <Sans size="2" weight="medium" color="black100">
+          Ways to buy
```

Comment:
Fixes inconsistency across artwork filters (and design system text idioms in general)